HW_TypeScript_Type_annotation_and_primitive_types.ts

```diff
@@ -0,0 +1,116 @@
+class School {
+    directions: any = [];
+
+    addDirection(direction: any) {
```

Comment:
Please do not forget to add explicit type annotation for the return function value.